staging/src/k8s.io/api/admissionregistration/v1alpha1/types.go

```diff
@@ -201,6 +201,12 @@ type ValidatingAdmissionPolicySpec struct {
 	// +listMapKey=name
 	// +optional
 	MatchConditions []MatchCondition `json:"matchConditions,omitempty" patchStrategy:"merge" patchMergeKey:"name" protobuf:"bytes,6,rep,name=matchConditions"`
+
+	// Variables contain definitions of variables that can be used in composition of other expressions.
```

Comment:
doc here isn't a full as the doc on the internal.  I think the details about referencing prior variable is important valuable.

Reply:
The docs in both places are de-sync'ed. I've checked these are now having same comments manually.